tests/test_settings.py

```diff
@@ -0,0 +1,21 @@
+from app.api.config import Settings
+
+
+def test_settings_read_correctly():
+    """Ensure that settings are read correctly from environment variables, with correct types and default values."""
+    settings = Settings()
```

Comment:
you mean that you're testing that the pytest.ini variables are correctly read here? Isn't that essentially testing pytest-env and pydantic-settings code? I.e. what would we have to mess up for this test to fail?

Reply:
This test checks that variables set in the environment (handled pytest.ini & pytest-env) are stored in corresponding internal variables in the app as the type we desire, and with correct default values if env vars are unset.

It does test a little bit of Pydantic settings behaviour (mainly the default value part, maybe), but I think it moreso serves as a test that we have defined our Settings model according to app expectations/needs. e.g., the app internally expects `NB_ENABLE_AUTH` to be a boolean. If we failed to specify that this variable should be a `bool` in `Settings`, that would be caught by this test since env vars are always read in as strings by default.